packages/js/product-editor/src/components/variations-table/variation-actions-menus/variation-quick-update-menu-item.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill, MenuItem } from '@wordpress/components';
+import { createElement, Fragment } from '@wordpress/element';
+import {
+	createOrderedChildren,
+	sortFillsByOrder,
+} from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import { MenuItemProps } from './types';
+import { QUICK_UPDATE, SINGLE_UPDATE } from './constants';
+
+const DEFAULT_ORDER = 20;
+
+const getGroupName = ( group?: string, isMultipleSelection?: boolean ) => {
+	const name = 'woocommerce-actions-menu-slot';
+	const nameSuffix = isMultipleSelection
+		? `_${ QUICK_UPDATE }`
+		: `_${ SINGLE_UPDATE }`;
+	return group ? `${ name }_${ group }${ nameSuffix }` : name;
+};
+
+const getMenuItem = ( children: React.ReactNode ) => (
+	<MenuItem>
+		<span>{ children }</span>
```

Comment:
Is there a reason for the span? I think we should do without it if we can... let's keep the DOM as tidy as possible. 😄

Reply:
I'd suggest adding a unit test for `onClick` too.

Reply:
> I'd suggest adding a unit test for onClick too.

Good idea! I added a few tests in the commit [dc7bccf](https://github.com/woocommerce/woocommerce/pull/43441/commits/dc7bccfc20be552a4dbd9bd308a11675f753ce2b)